codecov.yml

```diff
@@ -2,11 +2,13 @@ coverage:
   status:
     project:
       default:
-        informational: true
+        target: auto
```

Comment:
What does `auto` here mean?

Reply:
Target is the minimum allowed coverage. Value of `auto` means it's the coverage of the branch you're merging into. Alternatively you can put a percentage value to have an absolute value instead.

https://docs.codecov.com/docs/commit-status#target